pkgs/development/python-modules/complycube/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyhumps
+, pytestCheckHook
+, python
+, requests
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "complycube";
+  version = "1.1.6";
+  format = "wheel";
```

Comment:
```suggestion
  pyproject = true;
```
Can we use sdist?

Reply:
I'm not sure, they are just releasing the wheel

Reply:
No, they also provide source code.
https://pypi.org/project/complycube/#files

Reply:
here you go! :)